playbooks/byo/openshift-checks/health.yml

```diff
@@ -1,3 +1,10 @@
 ---
 - include: ../openshift-cluster/initialize_groups.yml
+  tags:
+  - always
+
+- include: ../../common/openshift-cluster/std_include.yml
```

Comment:
We have discussed where to put std_include.yml.  I originally wanted to put them in the `common` playbooks for just the reason you are mentioning, however as we walked through the componentization of the common playbooks, we found it was better to move that initialization up.  This would allow us to group different common components together without each of them having to run the initialization section over and over again.  I can explain it further if desired.